app/settings.py

```diff
@@ -24,7 +24,9 @@
     # Supported values: DynamoDB, InMemory, InFile.
     'BACKEND_STORAGE': 'DynamoDB',
     # Flask cache type, null means no caching.
-    'CACHE_TYPE': 'null'
+    'CACHE_TYPE': 'null',
+    # Set the connection pool size
```

Comment:
this comment is redundant

Reply:
ah realized that not all of these have comments. can delete.